modules/shared/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -37,14 +43,20 @@ object transaction {
     implicit def toAmount(fee: TransactionFee): Amount = Amount(fee.value)
   }
 
-  @derive(decoder, encoder, order, show)
+  @derive(decoder, encoder, show)
   @newtype
   case class TransactionOrdinal(value: NonNegLong) {
     def next: TransactionOrdinal = TransactionOrdinal(value |+| 1L)
   }
 
   object TransactionOrdinal {
     val first: TransactionOrdinal = TransactionOrdinal(1L)
+
+    implicit val transactionOrder: Order[TransactionOrdinal] = (x: TransactionOrdinal, y: TransactionOrdinal) =>
```

Comment:
I'd use just `transactionOrdinalOrder` or just `order` and `ordering` (as we know it's for TransactionOrdinal)